### PR TITLE
Fix product variable example appearing on 1 line

### PR DESCRIPTION
**Pull request by TylerCarberry**

### CODE REVIEW

File: using-mobile-extensions/adobe-analytics/product-variable.md [hunk 1]
```diff
@@ -12,14 +12,24 @@ To set the products variable, set a context data key to `&&products`, and set th
...
 #### Example <a id="example"></a>
 
-```text
-//create a context data dictionaryHashMap cdata = new HashMap<String, Object>();​// add products, a purchase id, a purchase context data key, and any other data you want to collect.// Note the special syntax for productscdata.put("&&products", ";Running Shoes;1;69.95,;Running Socks;10;29.99");cdata.put("myapp.purchase", "1");cdata.put("myapp.purchaseid", "1234567890");​// send the tracking call - use either a trackAction or TrackState call.// trackAction example:MobileCore.trackAction("purchase", cdata);// trackState example:MobileCore.trackState("Order Confirmation", cdata);
+```java
+//create a context data dictionary
+HashMap cdata = new HashMap<String, Object>();
+// add products, a purchase id, a purchase context data key, and any other data you want to collect.
+// Note the special syntax for products
+cdata.put("&&products", ";Running Shoes;1;69.95,;Running Socks;10;29.99");cdata.put("myapp.purchase", "1");
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-analytics/product-variable.md]: Can you put `cdata.put("myapp.purchase", "1");` on a new line as well?

File: using-mobile-extensions/adobe-analytics/product-variable.md [hunk 2]
```diff
@@ -12,14 +12,24 @@ To set the products variable, set a context data key to `&&products`, and set th
...
+cdata.put("myapp.purchaseid", "1234567890");
+// send the tracking call - use either a trackAction or TrackState call.
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-analytics/product-variable.md]: Use same format for both the function names `use either a trackAction or trackState call.`

File: using-mobile-extensions/adobe-analytics/product-variable.md [hunk 3]
```diff
@@ -12,14 +12,25 @@ To set the products variable, set a context data key to `&&products`, and set th
...
 cdata.put("&&products", "Category;Product;Quantity;Price[,Category;Product;Quantity;Price]");
 ```
 
 #### Example <a id="example"></a>
 
-```text
-//create a context data dictionaryHashMap cdata = new HashMap<String, Object>();​// add products, a purchase id, a purchase context data key, and any other data you want to collect.// Note the special syntax for productscdata.put("&&products", ";Running Shoes;1;69.95,;Running Socks;10;29.99");cdata.put("myapp.purchase", "1");cdata.put("myapp.purchaseid", "1234567890");​// send the tracking call - use either a trackAction or TrackState call.// trackAction example:MobileCore.trackAction("purchase", cdata);// trackState example:MobileCore.trackState("Order Confirmation", cdata);
+```java
+//create a context data dictionary
+HashMap cdata = new HashMap<String, Object>();
```
Comment (shalehaha) [inline using-mobile-extensions/adobe-analytics/product-variable.md:23]: change it to `<String, String>`

**Comment by TylerCarberry** [comment]
Feel free to commit directly to my branch!

**Pull request merged**